crates/common/certificate/Cargo.toml

```diff
@@ -17,7 +17,9 @@ reqwest = ["dep:reqwest"]
 anyhow = { workspace = true }
 camino = { workspace = true }
 rcgen = { workspace = true }
-reqwest = { workspace = true, optional = true }
+reqwest = { workspace = true, optional = true, features = [
+    "rustls-tls-native-roots",
```

Comment:
Not related specifically, but stuck this in as an additional commit while I noticed. If we try and run tests for a subset of dependencies, we try and build `certificate` without this feature, and that fails.